lib/phoenix/endpoint.ex

```diff
@@ -696,8 +696,16 @@ defmodule Phoenix.Endpoint do
       def static_path(path) do
         Phoenix.Config.cache(__MODULE__, :__phoenix_static__,
                              &Phoenix.Endpoint.Supervisor.static_path/1) <>
+        elem(static_lookup(path), 0)
+      end
+
+      def static_integrity(path) do
+        elem(static_lookup(path), 1)
+      end
+
+      def static_lookup(path) do
```

Comment:
Do we want to keep this one public or only private for now?

Reply:
I think it should be public. My plan after this was to use this function to create `script_tag` and `stylesheet_tag` functions in `phoenix_html` that auto include the integrity attribute. No sense in looking up the cache twice.

Reply:
Perfect.

Reply:
Perfect. Let's go ahead and document it too then!

Reply:
Will update